src/client/pydaos/daosdbm.py

```diff
@@ -135,9 +136,8 @@ def main():
 
     print('Kvs are {}'.format(','.join(sorted(my_kv.get_kv_list()))))
 
-    if True:
-        new_migrate(my_kv)
-        return
+    new_migrate(my_kv)
+    return
```

Comment:
This is very strange - returning in the middle of the function? The original issue was not really your bug but this is an error. Should this block be moved after the other checks? Or the stuff afterward should be removed?

Reply:
Thanks for the comment, bandit caught the issue of line 138 if True: as redundant statement, we might need to revisit the script for the rest of script planed to do.

Reply:
It seems like static analysis would tag this as unreachable code. Maybe @ashleypittman can comment on the intent. In this case I'd like the correct fix -- either changing the check to something meaningful, or removing the code afterward if it's not intended to be used.

Reply:
Most of this file is scheduled for removal in 1.2, it's not currently installed as part of the RPM.

Everything after L74 could be deleted.